README.md

```diff
@@ -61,3 +99,14 @@ work to the project under the project's open source license. Whether or not you
 explicitly, by submitting any copyrighted material via pull request, email, or other means you agree
 to license the material under the project's open source license and warrant that you have the legal
 authority to do so.
+For a more detailed step-by-step guide, please read
+[how to contribute to Alluxio](https://www.alluxio.org/docs/1.8/en/contributor/Contributor-Getting-Started.html).
+For new contributor, please take 2 [new contributor tasks](https://github.com/Alluxio/new-contributor-tasks).
```

Comment:
now that we're moving to issues and deprecating JIRA, should we remove this section?